package.json

```diff
@@ -35,6 +35,7 @@
     "oidc-client": "^1.10.1",
     "tiny-cookie": "^2.3.2",
     "vue-echarts": "^4.1.0",
+    "vue-gtag": "^1.6.2",
```

Comment:
I thought we woud opt for [vue-analytics](https://github.com/MatteoGabriele/vue-analytics) ?